website/docs/docs/collaborate/govern/project-dependencies.md

```diff
@@ -22,6 +22,7 @@ dbt Labs supports an expanded notion of `dependencies` across multiple dbt proje
 - For the upstream ("producer") project setup:
   - Configure models in upstream project with [`access: public`](/reference/resource-configs/access) and have at least one successful job run after defining `access`.
   - Define a deployment environment in the upstream project as [Production environment](/docs/deploy/deploy-environments#set-as-production-environment), ensuring at least one successful job run in that environment.
```

Comment:
i wonder if we can combine both messages in this bullet instead of introduce another bullet (new line added in line 25) so for example:

```suggestion
  - Define a deployment environment in the upstream project as [Production environment](/docs/deploy/deploy-environments#set-as-production-environment),  ensuring at least one successful deployment job run in that environment. Make sure the deployment job run generates an updated [manifest.json](/reference/artifacts/manifest-json) file, as this is contains necessary metadata information for downstream projects.
```